packages/app/src/sandbox/eval/presets/create-react-app/utils/reactPreset.ts

```diff
@@ -45,15 +45,27 @@ export const reactPreset = babelConfig => {
     aliases,
     {
       hasDotEnv: true,
-      processDependencies: async dependencies => {
+      processDependencies: async originalDeps => {
+        const deps = { ...originalDeps };
         if (
-          dependencies['react-dom'] &&
-          isMinimalReactDomVersion(dependencies['react-dom'], '16.9.0')
+          deps['react-dom'] &&
+          isMinimalReactDomVersion(deps['react-dom'], '16.9.0')
         ) {
-          return { ...dependencies, 'react-refresh': '0.9.0' };
+          deps['react-refresh'] = '0.9.0';
         }
 
-        return dependencies;
+        if (!deps['@babel/core']) {
+          deps['@babel/core'] = '^7.3.3';
+        }
+
+        if (!deps['@babel/runtime']) {
+          deps['@babel/runtime'] = '^7.3.4';
+        }
+
+        delete deps['babel-runtime'];
```

Comment:
This is to cleanup some deps we don't need, we can also start deprecating babel 6 if we want